image_class_info:
  image_folder_name: '/home/iarc/test_image/'
  object_classes:
    - juice
    - vitamin_drink
  image_num: 20
